spikesorters/kilosort/kilosort.py

```diff
@@ -94,20 +90,18 @@ def _setup_recording(self, recording, output_folder):
         assert isinstance(KilosortSorter.kilosort_path, str)
 
         # prepare electrode positions
-        electrode_dimensions = p['electrode_dimensions']
-        if electrode_dimensions is None:
-            electrode_dimensions = [0, 1]
-        if 'group' in recording.get_shared_channel_property_names():
+        if self.grouping_property == 'group' and 'group' in recording.get_shared_channel_property_names():
```

Comment:
Maybe here we could have a warning in case of:
self.grouping_property == 'group' but 'group' in recording.get_shared_channel_property_names().
or even better raise an error:
```python
if self.grouping_property == 'group' but 'group':
    assert recording.get_shared_channel_property_names()
   ....
```

Reply:
I think this should be handled by the base sorter

Reply:
https://github.com/SpikeInterface/spikesorters/pull/74/files#diff-7e5fe6dfe708847ac6c00d7ffea2427fR75